buildSrc/build.gradle.kts

```diff
@@ -155,11 +155,11 @@ fun Project.applyGroovyProjectConventions() {
     dependencies {
         compile(localGroovy())
         val spockGroovyVersion = groovy.lang.GroovySystem.getVersion().substring(0, 3)
-        testCompile("org.spockframework:spock-core:1.2-RC2-groovy-${spockGroovyVersion}") {
+        testCompile("org.spockframework:spock-core:1.2-RC3-groovy-${spockGroovyVersion}") {
```

Comment:
There's a couple more places where this needs upgraded.

Reply:
I was only able to find `BaseGradleImplDepsIntegrationTest`. Anywhere else?

Reply:
I just searched for `1.2-RC2`, but my IntelliJ stopped working, sorry. :)